incubator/hnc/e2e/tests/delete-anchor-crd_test.go

```diff
@@ -0,0 +1,74 @@
+package test
+
+import (
+	"os/exec"
+	"os"
+	"time"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Delete-anchor-crd", func() {
+	It("should create parent and deletable child, and delete the CRD", func() {
+		GinkgoT().Log(time.Now())
+
+		// create a namespace delete-crd-parent
+		err := runcmd("kubectl create ns delete-crd-parent")
+		Expect(err).Should(BeNil())
```

Comment:
Much better! But the line `Expect(err).Should(BeNil())` is repeated every time - why not include that in `runcmd` as well?

Reply:
I was thinking what if we are actually expecting an error some time